src/store/useTemplatesStore.ts

```diff
@@ -57,12 +76,10 @@ const store = (set: any, get: any): TemplateState => ({
 			error?: string;
 			templateMap?: Record<string, TemplateMetadataWithCategory>;
 		}> => {
-			const processTemplateCard = async (
-				cardWithFiles: TemplateCardWithFiles,
-				categoryName: string,
-				templateStorage: TemplateStorageService
-			) => {
-				await templateStorage.storeTemplateFiles(cardWithFiles.assetDirectory, cardWithFiles.files);
+			const processTemplateCard = async (cardWithFiles: TemplateCardWithFiles, categoryName: string) => {
```

Comment:
```suggestion
			const processTemplateCard = async (cardWithFiles, categoryName) => {
```

Reply:
I'm getting: Parameter 'cardWithFiles' implicitly has an 'any' type.ts(7006)

This is internal function so it's legit that it will have types.